### PR TITLE
Update Cliff's AboutUs

### PR DESCRIPTION
**Pull request by Domokunx**

### CODE REVIEW

**Review by raageth — COMMENTED** [review]
LGTM

**Review by sayglenn — APPROVED** [review]
LGTM!

File: docs/team/cliff.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ title: John Doe's Project Portfolio Page
```
Comment (raageth) [inline docs/team/cliff.md:5]: Should the title (line 3) be your name instead of John Doe?

Comment (Domokunx) [reply docs/team/cliff.md:5]: will fix

**Pull request merged**